contracts/strategies/curve/ankrETH/CRVStrategyAnkrETH.sol

```diff
@@ -0,0 +1,258 @@
+pragma solidity 0.5.16;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/math/Math.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "../../../base/interface/curve/Gauge.sol";
+import "../../../base/interface/curve/ICurveAnkrETHDeposit.sol";
+import "../../../base/interface/IStrategy.sol";
+import "../../../base/interface/IVault.sol";
+import "../../../base/interface/ILiquidatorRegistry.sol";
+import "../../../base/interface/ILiquidator.sol";
+import "../../../base/interface/weth/Weth9.sol";
+
+import "../../../base/StrategyBaseUL.sol";
+
+import "hardhat/console.sol";
```

Comment:
This may be removed, right? (debugging-use only)

Reply:
Indeed, will remove